doc/releases/changelog-dev.md

```diff
@@ -255,6 +255,7 @@
     [(#7347)](https://github.com/PennyLaneAI/pennylane/pull/7347)
     [(#7352)](https://github.com/PennyLaneAI/pennylane/pull/7352)
     [(#7362)](https://github.com/PennyLaneAI/pennylane/pull/7362)
+    [(#7499)](https://github.com/PennyLaneAI/pennylane/pull/7499)
```

Comment:
Shall we have a specific line to summarize this change?

Reply:
No, the changelog reflects changes in this version compared to the last version. It does not make sense to say that we removed something in the changelog that didn't exist in the previous version to begin with.